src/components/Form/story.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Form from '.';
+import Readme from './README.md';
+
+storiesOf('Uncategorized/Form', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Form
+        initialValues={{ nameInput: 'james r. cat' }}
+        // eslint-disable-next-line no-alert
+        onSubmit={(formValues) => alert(JSON.stringify(formValues))}
+        validationSchema={{
+          nameInput: {
+            message: 'No empty strings',
+            validate: (value) => value !== '',
+          },
+        }}
+      >
+        {({
+          formState,
+          formValues,
+          handleChange,
+          handleBlur,
+          handleCustom,
+        }) => (
+          <>
+            <label htmlFor="nameInput">
+              Name
+              <br />
+              <input
+                type="text"
+                name="nameInput"
+                value={formValues.nameInput}
```

Comment:
This would be crazy:
Having the user enter `name="foo"` _and_ `formValues.foo` feels redundant.
Would it make sense to iterate the children/dom elements and apply values and event handlers that way? I'm not sure it would be feasible or fast, but the interface would sure be nice if you could just add `<input type="text" name="myInput" />` and everything else was automatic.

We might be able to do something like that with an `EasyInput` component and the React Context API.

Reply:
```<Form initialValues={{
  foo: ""
}}>
<EasyInput name="foo" type="text" />
</Form>
```